core/node/consistency_checker/src/lib.rs

```diff
@@ -777,7 +737,6 @@ impl ConsistencyChecker {
                     self.event_handler
                         .report_inconsistent_batch(batch_number, &err);
                     match &self.l1_data_mismatch_behavior {
-                        #[cfg(test)]
                         L1DataMismatchBehavior::Bail => {
```

Comment:
do we need to handle this bail somehow similarly to how you handled the error from gateway migrator?

Reply:
So, we don't handle it ;) 
we have a find function for convenience. I Don't think we have to handle it somehow else